### PR TITLE
cilium, bigtcp: Add max gso/gro rates to sysdump

### PR DESCRIPTION
**Pull request by borkmann**
See commts.

### CODE REVIEW

**Comment by borkmann** [comment]
Cc @haiyuewa, fyi, might be interesting for you as well. :)

File: pkg/client/client.go [hunk 1]
```diff
@@ -391,15 +391,23 @@ func FormatStatusResponse(w io.Writer, sr *models.StatusResponse, sd StatusDetai
...
 
 	if sr.IPV4BigTCP != nil {
-		status := "Enabled"
+		max := fmt.Sprintf("[%d, %d]", sr.IPV4BigTCP.MaxGRO, sr.IPV4BigTCP.MaxGSO)
+		if sr.IPV4BigTCP.MaxGRO == sr.IPV4BigTCP.MaxGSO {
+			max = fmt.Sprintf("[%d]", sr.IPV4BigTCP.MaxGSO)
+		}
+		status := fmt.Sprintf("Enabled\t%s", max)
 		if !sr.IPV4BigTCP.Enabled {
 			status = "Disabled"
```
Comment (aspsk) [inline pkg/client/client.go]: Looks like if `!sr.IPV4BigTCP.Enabled`, then we do three extra Sprintf's at this point. We could optimize this to one `fmt.Fprintf` per case. (Not a big deal though, as this is just `cilium status`, so ack.)

**Review by aspsk — APPROVED** [review]

**Pull request merged**